.github/workflows/config/spelling_allowlist.txt

```diff
@@ -235,6 +235,7 @@ fmt
 fourier
 frontend
 frontends
+grovers
```

Comment:
Why did you need to add this to the spellcheck? Did you try without adding that? If it is because of the index in applications.rst, it's a bit suboptimal, since we wouldn't want this spelling to be in text (should be Grover's there). 
Please use `|:spellcheck-disable:| text |:spellcheck-enable:|`  instead and remove it from here.